.github/pull_request_template.yml

```diff
@@ -0,0 +1,8 @@
+
+
+
+---
+
+🚨⚠️ This will be deployed automatically all the way to production when you click merge ⚠️🚨
+
+See [docs/deploying.md](docs/deploying.md) for notes and caveats about this process.
```

Comment:
I know I wouldn't click on this link to read any additional docs as I would assume it is generic stuff from the wiki for a process I'm already familiar with that is shared across all our continuously deployed apps.

If there are caveats that people need to be aware of when merging a PR I suggest either we show it inline to them as part of the code they are changing or that it goes in this pull request template. Given how short the docs are below as well, moving it in to the pull request template wouldn't be too long I don't think.

Reply:
> I know I wouldn't click on this link to read any additional docs

I would the first time, but not subsequent times. That's probably the most value we can expect to get out of this. I've changed "process" to "app", though. Would that be more of a trigger for you?

I'm not a fan of PR templates: they usually add noise. However, it does seem like the best mitigation for the risk here. I'd prefer to keep it as short as possible, by avoiding low-level guidance that could grow. I agree it's not _that_ long right now, but I can imagine it getting longer e.g. more pointers, or more caveats coming along. I don't want to have to scroll past a load of boilerplate every time I read someone's PR - [here's a past example I regret](https://github.com/alphagov/smokey/pull/695).

Since these are both subjective views, maybe we can just try it and see if someone goes "I would have done that testing if the warning had been more in-my-face 😄". What do you think?

Reply:
I actually agree with David here, I wouldn't read those docs either, but I would read a PR template. So it would be great to have something in the PR template, even if it's a pointer to the docs, like: "Caution, when you hit merge, this will deploy your changes all the way to prod, for more info about deployment process for this app see the README".

Reply:
As you say, it is subjective and not something I mind too much given the short and low cost nature of having this extra documentation.

I think I'd still suggest putting it all in the PR template but will approve this anyway because I don't think it is a high enough impact change to warrant too lengthy a discussion for something that is subjective and we have little evidence for either way.

Reply:
> for more info about deployment process for this app see the README

@CrystalPea I think the latest template should do just that 👌.

> I'd still suggest putting it all in the PR template but will approve this anyway

I get you. Given your comments I'd do that if I started again, but it's not worth it now.